webapp/channels/src/utils/message_html_to_component.tsx

```diff
@@ -29,25 +57,25 @@ import PostEditedIndicator from 'components/post_view/post_edited_indicator';
  * - hasPluginTooltips - If specified, the LinkTooltip component is placed inside links. Defaults to false.
  * - channelId = If specified, to be passed along to ProfilePopover via AtMention
  */
-export function messageHtmlToComponent(html, options = {}) {
+export function messageHtmlToComponent(html: any, options: Options = {}) {
     if (!html) {
         return null;
     }
 
-    const parser = new Parser();
-    const processNodeDefinitions = new ProcessNodeDefinitions(React);
+    const parser = new (Parser as any)();
+    const processNodeDefinitions = new (ProcessNodeDefinitions as any)();
```

Comment:
Why are these any needed?

Reply:
The original library doesn't have defined the constructor, but we need to use the `new` so typescript complains about it. Again a problem in the original library.